README.md

```diff
@@ -80,63 +76,63 @@ noteCount                      - int, note events count.
         SliderTail = 5, 
         BurstSliderHead = 6, 
         BurstSliderElement = 7
-  eventTime                    - float, song time of event 
-  spawnTime                    - float, spawn time of note
-  eventType                    - int, good = 0,bad = 1,miss = 2,bomb = 3
-  {                            - Cut info structure (only for Good and Bad!)
-    bool speedOK;
-    bool directionOK;
-    bool saberTypeOK;
-    bool wasCutTooSoon;
-    float saberSpeed;
-    Vector3 saberDir;
-    int saberType;
-    float timeDeviation;
-    float cutDirDeviation;
-    Vector3 cutPoint;
-    Vector3 cutNormal;
-    float cutDistanceToCenter;
-    float cutAngle;
-    float beforeCutRating;
-    float afterCutRating;
+  eventTime                    - float, song time of event.
+  spawnTime                    - float, spawn time of note.
+  eventType                    - int, good = 0, bad = 1, miss = 2, bomb = 3.
+  {                            - Cut info structure (only for Good and Bad!).
+    speedOK                    - bool, note was hit at passable speed.
+    directionOK                - bool, note was hit with correct direction.
+    saberTypeOK                - bool, note was hit with correct saber.
+    wasCutTooSoon              - bool, note was cut too soon.
+    saberSpeed                 - float, speed of the saber.
+    { x, y, z }                - 3 floats, direction of the saber.
```

Comment:
Let's preserve the name of the property, as it's more specific and corresponds to the game name
`{ x, y, z }                - 3 floats, saberDir, direction of the saber.`